app/javascript/components/recordings/Recordings.jsx

```diff
@@ -44,6 +46,9 @@ export default function Recordings() {
                       <div key={format.id}> {format.recording_type} </div>
                     ))}
                   </td>
+                  <td>
+                    <FontAwesomeIcon className="" icon={faTrashCan} size="lg" onClick={() => handleDeleteRecording({ recordId: recording.id })} />
```

Comment:
I think this should pop a modal up to be safe

Reply:
Heads up we have a Modal template in components/shared.

I did a "font-awesome" button variant as well if you can check it out in #3447 application.scss if you want (not merged yet).

![image](https://user-images.githubusercontent.com/43917914/169385896-26a3a08a-160f-48ce-9c7c-0a47ab8c99c7.png)

Reply:
Thanks for the tip Samuel helped a lot.